app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
list of what?

Please add additional info for annotation 
fix here and  other places